libs/platform/src/lib/components/form/form-group/form-field/form-field.component.ts

```diff
@@ -166,7 +172,8 @@ export class FormFieldComponent
     protected _destroyed = new Subject<void>();
 
     constructor(private _cd: ChangeDetectorRef) {
-        this.formControl = new FormControl();
+        // provides capability to make a field disabled. useful in reactive form approach.
+        this.formControl = new FormControl({ value: null, disabled: this.disabled });
```

Comment:
Can you fix formatting, is i see indentation in constructor is different then on ngOnInit()